Model/Data/Validation.php

```diff
@@ -109,8 +109,30 @@ public function validate(
             return $this;
         }
 
+        if (!$quote->getReservedOrderId()) {
+            $this->logger->addRvvupError(
+                'Rvvup Quote missing reserved order id',
+                null,
+                $rvvupId,
+                $lastTransactionId,
+                $origin
```

Comment:
this is setting origin to: $magentoOrderId, so should be:
```suggestion
null, 
                $origin
```

Reply:
quick one, can we just sort the formatting on this please